www/_posts/2017-04-21-cordova-7.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+author:
+    name: Audrey So
+    url: https://twitter.com/aud_rey_so
+title:  "Apache Cordova 7.0.0"
+categories: news
+tags: news releases
+---
+
+We are happy to announce that `Apache Cordova 7.0.0` has been released!
+
+Most notable changes include:
+* If a `package.json` does not exist in your project, it will be auto-created for you when `cordova prepare` is called.
+* When adding a platform or plugin, it will automatically save that platform or plugin to your `config.xml` and `package.json`. Details about platform and plugin versions are also automatically saved in `config.xml` and `package.json`. The `--save` flag is no longer required to save. Use `--nosave` to prevent saving to `config.xml` or `package.json`.
+* Fetch is now the default setting for fetching platforms. Fetch uses your system `npm` to `npm install` modules into your project. The `--fetch` flag is no longer required. Use the `--nofetch` flag to use the older cordova fetching method (pre `cordova@7`) and prevent using `npm install` to do the fetching.
```

Comment:
Hmmm. To me you're referring to the feature named "fetch", not the setting (although you do use the word "setting" -- I think I would use "method"). So:

"Fetch is now the default method for fetching platforms. Fetch uses your system...."

You could reword this a bit to avoid the issue entirely, but I think this is clear enough without being overly verbose.